### PR TITLE
Adding option to deploy atom with k8s

### PR DESCRIPTION
**Pull request by baltekgajda**
Using ansible playbook - deploy.yaml: https://review.opencontrail.org/#/c/46176/

### CODE REVIEW

File: provision/kubernetes/kubernetes.tf [hunk 1]
```diff
@@ -171,6 +179,23 @@ resource "openstack_compute_floatingip_associate_v2" "floatip_1" {
...
+      password    = ""
+      agent       = "false"
+      host        = "${openstack_networking_floatingip_v2.floatip_1.address}"
+      private_key = "${file(var.ssh_private_key)}"
+      timeout     = "5m"
+    }
+
+    inline = [
+      "cd ${local.contrail_path}",
+      "ansible-playbook -e contrail_type=${var.contrail_type} -e contrail_path=${local.contrail_path} playbooks/contrail-go/deploy.yaml",
```
Comment (kravvcuCL) [inline provision/kubernetes/kubernetes.tf]: cant this be put in the previous remote-exec before restarting the VM?

File: provision/kubernetes/variables.tf [hunk 2]
```diff
@@ -51,3 +51,12 @@ variable "flavor" {
   default = "@FLAVOR"
 }
 
+variable "contrail_type" {
+  default = "vnc_api"
+}
+
+variable "patchsetRef" {
```
Comment (kravvcuCL) [inline provision/kubernetes/variables.tf]: rather patchset_ref for compliancy with other vars naming

File: provision/kubernetes/variables.tf [hunk 3]
```diff
@@ -51,3 +51,12 @@ variable "flavor" {
...
+  default = ""
```
Comment (buoto) [inline provision/kubernetes/variables.tf]: Maybe default patchset should be `master` for clarity?

File: provision/kubernetes/kubernetes.tf (hunk 1 above)
Comment (baltekgajda) [reply provision/kubernetes/kubernetes.tf]: I moved it to previous remote-exec

File: provision/kubernetes/variables.tf (hunk 3 above)
Comment (baltekgajda) [reply provision/kubernetes/variables.tf]: Done

File: provision/kubernetes/variables.tf (hunk 2 above)
Comment (baltekgajda) [reply provision/kubernetes/variables.tf]: Done

**Review by kravvcuCL — APPROVED** [review]

**Pull request merged**